client/src/Movies/SavedList.js

```diff
@@ -12,7 +13,7 @@ export default class SavedList extends Component {
         {this.props.list.map(movie => (
           <span className="saved-movie">{movie.title}</span>
         ))}
-        <div className="home-button">Home</div>
+        <Link to= {`/`} className="home-button">Home</Link>
```

Comment:
You can actually write your `to` without having to use curly brackets like so: `to='/'`